deno.json

```diff
@@ -0,0 +1,12 @@
+{
+  "lint": {
+    "files": {
+      "include": ["src/", "./mod.ts", "./rollup.config.ts"]
+    }
+  },
+  "fmt": {
```

Comment:
@mathe42 can we use tabs here? And then re-fmt the PR?

```js
"options": {
      "useTabs": true,
      "lineWidth": 80,
      "indentWidth": 4,
      "singleQuote": false,
 }
```

Reply:
Done!